### DISCUSSION TITLE
Close dropdown after clicked on item

### DISCUSSION THREAD

**Discussion by MuslemRahimi** [saadeghi/daisyui#1604] [Q&A] [answered]
Hi,

I want that the dropdown should vanish as soon as i click one of the items. 

Is it possible to modify the code accordingly?

```
<div class="dropdown">
  <label tabindex="0" class="btn m-1">Click</label>
  <ul tabindex="0" class="dropdown-content menu p-2 shadow bg-base-100 rounded-box w-52">
    <li><a>Item 1</a></li>
    <li><a>Item 2</a></li>
  </ul>
</div>

**Comment by saadeghi** [comment, marked as answer]
Currently dropdown works with CSS focus. So as long as it is focused, it will stay open. You can remove the focus in many ways depending on what JS library you're using. Read this discussion for more info: https://github.com/saadeghi/daisyui/issues/201

Let me know if you have any questions.

**Comment by FARZINzx** [comment]
`const handleDropDownItemClick = () => {
    const activeElement = document.activeElement as HTMLElement;
    if (activeElement && activeElement !== document.body) {
      activeElement.blur(); // Remove focus from the currently focused element
    }
  }`
